src/MediaPlayer/MediaPlayer.cs

```diff
@@ -319,6 +324,7 @@ public enum MPMediaLibraryAuthorizationStatus : nint {
 
 	[Mac (10,12,2)]
 	[iOS (10,0)]
+	[TV (10,0)]
```

Comment:
@chamons can you double check xtro results ? This should have been reported for tvOS